recipes/pyutilib/meta.yaml

```diff
@@ -87,8 +87,6 @@ test:
     # entry points work.
 
     - test.pyutilib --help
-    - lbin --help
-    - lpython --help
```

Comment:
The lbin and lpython commands do not support '--help' options.

That should be fixed, but it's not in this release of PyUtilib.

--Bill

On Tue, Apr 12, 2016 at 3:13 PM, Filipe <notifications@github.com> wrote:

> In recipes/pyutilib/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/331#discussion_r59436001>
> :
>
> > @@ -87,8 +87,6 @@ test:
> >      # entry points work.
> >
> >      - test.pyutilib --help
> > -    - lbin --help
> > -    - lpython --help
>
> What was the failure?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/331/files/79e192f0ac0d17b9554f605598a57a1d206c4aaf..2147bdd76e61312dbfd8266a2a0087d85a579220#r59436001>
>